packages/flutter_tools/test/src/context.dart

```diff
@@ -234,3 +236,5 @@ class _MockUsageTimer implements UsageTimer {
 }
 
 class MockFlutterVersion extends Mock implements FlutterVersion {}
+
+class MockClock extends Mock implements Clock {}
```

Comment:
It's such a simple interface, it may be worth writing a `FakeClock` and saving your tests the effort of mocking...  It could have a setter for setting the current time, and it'd increment the value every time `now()` was called.